packages/compass-telemetry/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export { createTrack } from './ipc-track';
+export { type TrackFunction } from './track';
+export { useTrackOnChange } from './hooks';
```

Comment:
This should be exported from the provider instead to make sure that you're not pulling electron specific dependencies in the shared code